### DISCUSSION TITLE
Bug when selecting hour or minute in Qtimer

### DISCUSSION THREAD

**Discussion by thiagoreri** [quasarframework/quasar#16541] [General - Components / Directives / etc]
When selecting the hour or minute in QTimer, the component is setting incorrect values

**Comment by pdanpdan** [comment]
Any details on this?

**Comment by thiagoreri** [comment]
For example when I click on a minute, it sets another

**Comment by thiagoreri** [comment]
![image](https://github.com/quasarframework/quasar/assets/8278951/c7c46d4d-fc8f-46ab-ab0b-7f4d5a7e4c2b)

When I click on the red square, it stays where it is

**Comment by thiagoreri** [comment]
`<q-time v-model="model"/>`

Look, I put it in without anything to see if it was a configuration issue, but this still happens.

**Comment by thiagoreri** [comment]
Does anyone have a possibility

**Comment by rstoenescu** [comment]
Please make a [codepen](https://codepen.quasar.dev) with your issue. With the information that you supplied so far it is not reproducible. Also, make sure that you are using the latest Quasar UI version.
